indy_client/test/cli/test_node_suspension.py

```diff
@@ -1,35 +1,22 @@
 from plenum.common.util import randomString, hexToFriendly
-from plenum.common.constants import SERVICES, VALIDATOR, TARGET_NYM, DATA
+from plenum.common.constants import SERVICES, TARGET_NYM, DATA
 from plenum.common.txn_util import get_payload_data
 
-from plenum.test.pool_transactions.helper import sdk_add_new_nym, sdk_add_new_node, demote_node
-from indy_client.test.cli.helper import doSendNodeCmd
+from plenum.test.pool_transactions.helper import sdk_add_new_nym, sdk_add_new_node, demote_node, promote_node
 
 
-def testSuspendNode(be, do, trusteeCli, newNodeAdded):
+def testSuspendNode(looper, sdk_pool_handle, sdk_wallet_trustee, newNodeAdded):
```

Comment:
Please move the test into indy-node folder

Reply:
I thought to do this in the next PR. And this one mainly for deleting. Tell me, if we need to do that in this PR.